internal/events/loop.go

```diff
@@ -88,82 +83,84 @@ func (el *EventLoop) handleEvent(ctx context.Context, event interface{}) error {
 		return err
 	}
 
-	el.processChangesAndStatusUpdates(ctx, changes, updates)
+	changed, conf, statuses := el.processor.Process()
+	if !changed {
+		return nil
+	}
+
+	err = el.updateNginx(ctx, conf)
+	if err != nil {
+		el.logger.Error(err, "Failed to update NGINX configuration")
+	}
+
+	// FIXME(pleshakov) Update resource statuses instead of printing to stdout
+	for name, s := range statuses.ListenerStatuses {
+		fmt.Printf("Listener %q, Statuses: %v\n", name, s)
+	}
+	for nsname, s := range statuses.HTTPRouteStatuses {
+		fmt.Printf("HTTPRoute %q, Statuses: %v\n", nsname.String(), s)
```

Comment:
Since `nsname` implements the `String()` method you should be able to print it like this:
`fmt.Printf("%q", nsname)`